tools/harvest_source_import/tests/test_import_sources.py

```diff
@@ -11,12 +11,15 @@ def test_load_from_url():
 
     ckan = RemoteCKAN(url='https://catalog.data.gov')
     ckan.set_destination(ckan_url='http://ckan:5000',
-                         ckan_api_key='7564fcf7-1e79-4e03-a052-f94979051770')
+                         ckan_api_key='783b1397-3a9f-4ffe-81c3-c67cc85ef5f1')
```

Comment:
Curious: why did the API key change?

Reply:
A way to avoid require a full CKAN running is to save the request responses with Pytest VCR cassettes.
Each API key is the valid API keys while I test this (changes every time I clean my environment). I need to update to fit the exact request. [Here](https://github.com/GSA/catalog.data.gov/blob/42_expand_imported_sources_info/tools/harvest_source_import/tests/cassettes/test_load_from_url.yaml#L989:L989) you can see where this API KEY is used.

Reply:
That's what I thought ... but shouldn't the changed pytest cassette be part of this pull request, then?